example/ofp_netwrap_crt/netwrap_common.c

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2020.
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+#include "netwrap_common.h"
+
+ofp_global_param_t ofp_global_params = {0};
+odp_bool_t netwrap_constructor_called;      /* = 0 */
```

Comment:
This looks a lot like something that can be accessed concurrently by multiple threads. If so, synchronization is needed to get rid of the data race. The variable could be changed to odp_atomic_u32_t and written using odp_atomic_store_rel_u32( ) and read using odp_atomic_load_acq_u32().